sections/04_study.md

```diff
@@ -13,16 +13,16 @@ microbial communities, and predict protein structures (which relates to the
 problems discussed in the drug development section). In addition, unsupervised
 deep learning has enormous potential for discovering novel cellular states from
 gene expression, fluorescence microscopy, and other types of data that
-ultimately prove to be clinically relevant.
+may ultimately prove to be clinically relevant.
 
 Progress has been rapid in genomics and imaging, fields where important tasks
 are readily adapted to well-established deep learning paradigms.  One
 dimensional convolutional and recurrent neural networks are well-suited for
-important problems in protein binding of DNA and RNA, epigenomics, and RNA
+problems in protein binding of DNA and RNA, epigenomics, and RNA
 splicing.  Two dimensional CNNs are ideal for segmentation, feature extraction,
 and  classification in fluorescence microscopy images
 [@doi:10.3109/10409238.2015.1135868]. Other areas, such as cellular signaling,
-are biologically important but studied less-frequently to date
+are biologically important but studied less-frequently to date, with some exceptions
 [@tag:Chen2015_trans_species].  This may be a consequence of data limitations or
```

Comment:
This reuse was intentional in the original draft expressed the idea "data: it may be limited or we may not be using it optimally."

Do you have a suggested rephrasing?